src/pages/guides/getting_started/local_development/mcp_server.md

```diff
@@ -0,0 +1,251 @@
+        "-registry=https://artifactory.corp.adobe.com/artifactory/api/npm/npm-adobe-release/",
+        "@adobe/ccweb-add-on-dev-mcp-server"
+      ]
+    }
+  }
+}
+```
+
+## Available Tools
+
+The Adobe Express Add-on MCP Server provides two main tools for development assistance:
+
+### 1. Documentation Retrieval (`get_relevant_documentations`)
+
+This tool searches and retrieves relevant documentation from the Adobe Express add-on developer documentation.
+
+**Usage Examples:**
+
+- "How do I create text in Adobe Express?"
+- "What are the steps for drag and drop functionality?"
+- "How does the Document API work?"
+- "Show me examples of using the color picker"
+
+**Features:**
+
+- Semantic search across all Adobe Express add-on documentation
+- Real-time access to the latest guides, tutorials, and references
+- Contextual examples and code snippets
+- Links to related documentation sections
+
+### 2. TypeScript Definitions (`get_typedefinitions`)
+
+Provides complete TypeScript definitions for Adobe Express add-on APIs, enabling better code completion and type checking.
+
+**Available API Types:**
+
+#### `iframe-ui`
+
+Complete TypeScript definitions for building add-on UI panels:
+
+- **UI components**: Access to Adobe Express UI elements
+- **Event handling**: Respond to user interactions and application events
+- **Document operations**: Import/export content, create renditions
+- **OAuth integration**: Handle user authentication
+- **Storage management**: Persist add-on data
+
+Key interfaces:
+
+- `AddOnSDKAPI` - Main SDK entry point
+- `Application` - Access to Express application features
+- `Document` - Document-level operations
+- `UI` - User interface management
+
+#### `express-document-sdk`
+
+Complete TypeScript definitions for the Adobe Express Document APIs, including:
+
+- **Document manipulation**: Create, modify, and manage Express documents
+- **Content creation**: Add text, shapes, images, and other elements
+- **Styling and effects**: Apply fills, strokes, colors, and visual effects
+- **Layout and positioning**: Control element placement and transformations
+- **Media handling**: Work with images, videos, and audio content
+
+Key classes and interfaces:
+
+- `Editor` - Main entry point for document manipulation
+- `Node` and subclasses - Represent document elements
+- `Color`, `Fill`, `Stroke` - Styling properties
+- `TextNode`, `RectangleNode`, `EllipseNode` - Specific element types
+
+#### `add-on-sdk-document-sandbox`
+
+TypeScript definitions for the [Document Sandbox](../../../references/document-sandbox/index.md) runtime environment:
+
+- **Runtime management**: Handle sandbox execution environment
+- **Communication APIs**: Bridge between UI and document sandbox
+- **Type safety**: Ensure proper data exchange between contexts
+
+Key components:
+
+- `AddOnDocumentSandboxSdk` - Main SDK interface
+- `Runtime` - Sandbox execution environment
+- Communication proxies and type definitions
+
+## Usage Examples
```

Comment:
Lets trim this down and only specify a few examples such as 
"implement color picker in my add-on" or "implement drag and drop on this element"